src/components/dropdown/dropdown.tsx

```diff
@@ -20,6 +21,7 @@ export interface DropdownProps<T> {
 
 export interface DropdownValueProps<T> {
   value?: T;
+  publicKeyHash?: string;
```

Comment:
it's better to pass `onLongPress` handler as a prop and move `copyStringToClipboard(publicKeyHash)` into `CurrentAccountDropdown` component

Reply:
done)